engine/Dockerfile

```diff
@@ -28,7 +28,8 @@ RUN mkdir -p /tmp/silk_profiles;
 RUN chown -R 1000:2000 /tmp/silk_profiles
 
 FROM base AS dev
-RUN apt-get install -y sqlite3 default-mysql-client postgresql-client
```

Comment:
this change should be reverted prior to merging. This is a temporary workaround to this issue when building the engine docker image:
```bash
→ E: Failed to fetch http://deb.debian.org/debian-security/pool/updates/main/m/mariadb-10.3/mariadb-client-core-10.3_10.3.36-0+deb10u2_arm64.deb  404  Not Found [IP: 199.232.150.132 80]
       → E: Unable to fetch some archives, maybe run apt-get update or try with --fix-missing?
     
     ERROR IN: [dev 1/1] RUN apt-get install -y sqlite3 default-mysql-client [redacted secret helm-testing-postgresql:postgres-password]ql-client

Build Failed: ImageBuild: executor failed running [/bin/sh -c apt-get install -y sqlite3 default-mysql-client [redacted secret helm-testing-postgresql:postgres-password]ql-client]: exit code: 100
```